themes/default/content/blog/deploy-applications-with-aws-app-runner/index.md

```diff
@@ -107,42 +106,58 @@ Where index.html is simply:
 To push the image, we simply use Pulumi’s Image component resource to build and push. We’ll need our ECR credentials, which we can obtain via a simple function call and pass that information as an argument to our `Image` resource:
 
 ```go
-   ecrCredentials, err := ecr.GetAuthorizationToken(ctx, &ecr.GetAuthorizationTokenArgs{
-       RegistryId: nil,
-   }, nil)
-   if err != nil {
-       return err
-   }
-
-   // Build and push our image into the ECR repo.
-   dockerImage, err := docker.NewImage(ctx, "sampleapp", &docker.ImageArgs{
-       ImageName: repo.RepositoryUrl,
-       Build: docker.DockerBuildArgs{
-           Context: pulumi.String("./app"),
-       },
-       Registry: &docker.ImageRegistryArgs{
-           Server:   repo.RepositoryUrl,
-           Username: pulumi.String(ecrCredentials.UserName),
-           Password: pulumi.String(ecrCredentials.Password),
-       },
-   })
-   if err != nil {
-       return err
-   }
+ecrCredentials, err := ecr.GetAuthorizationToken(ctx, &ecr.GetAuthorizationTokenArgs{
+    RegistryId: nil,
+}, nil)
+if err != nil {
+    return err
+}
+
+// Build and push our image into the ECR repo.
+dockerImage, err := docker.NewImage(ctx, "sampleapp", &docker.ImageArgs{
+    ImageName: repo.RepositoryUrl,
+    Build: docker.DockerBuildArgs{
+        Context: pulumi.String("./app"),
```

Comment:
We should mention in the post that the Dockerfile and index.html need to live in this folder. Otherwise, the update fails.

Reply:
Ah, thanks for pointing that out. Will update the text to make this clear.